pkg/util/azureclient/mgmt/compute/virtualmachinesscalesets_addons.go

```diff
@@ -40,3 +41,11 @@ func (c *virtualMachineScaleSetsClient) List(ctx context.Context, resourceGroupN
 	}
 	return scaleSets, nil
 }
+
+func (c *virtualMachineScaleSetsClient) Get(ctx context.Context, resourceGroupName string, vmScaleSetName string) (mgmtcompute.VirtualMachineScaleSet, error) {
+	result, err := c.VirtualMachineScaleSetsClient.Get(ctx, resourceGroupName, vmScaleSetName)
+	if err != nil {
+		return mgmtcompute.VirtualMachineScaleSet{}, err
+	}
+	return result, nil
+}
```

Comment:
No need to wrap this: upstram implementation which is embeded) matches this interface. See the comment about `VirtualMachineScaleSetsClient` vs `VirtualMachineScaleSetsClientAddons` above.

```suggestion
```